src/pages/PassportRedemption/Passport.tsx

```diff
@@ -79,36 +79,40 @@ const Passport = (props: Props) => {
   }, [id]);
 
   const createTicketRows = (tickets) => {
-    // make a temp ticket that sorts the tickets by sponsor seller, then redemption date
-    const tempTickets = [...tickets];
-    const sortedTickets = tempTickets
-      .sort((a, b) => {
-        return b.sponsor_seller_id - a.sponsor_seller_id;
-      })
-      .sort((a, b) => {
-        const dateA = a.redeemed_at ? Date.parse(a.redeemed_at) : 0;
-        const dateB = b.redeemed_at ? Date.parse(b.redeemed_at) : 0;
-        return dateB - dateA;
-      });
+    
     let rows: any[] = [];
 
-    // group the entries by sponsor_seller_id,
-    const groupedTickets = groupBy(sortedTickets, 'sponsor_seller_id');
-    const newEntries = Object.keys(groupedTickets);
+    if (tickets.length > 0) {
+      // make a temp ticket that sorts the tickets by sponsor seller, then redemption date
+      let tempTickets = [...tickets]; 
+      const sortedTickets = tempTickets
+        .sort((a, b) => {
+          return b.sponsor_seller_id - a.sponsor_seller_id;
+        })
+        .sort((a, b) => {
+          const dateA = a.redeemed_at ? Date.parse(a.redeemed_at) : 0;
+          const dateB = b.redeemed_at ? Date.parse(b.redeemed_at) : 0;
+          return dateB - dateA;
+        });
```

Comment:
I think this would be a safer and more concise sorting algo:
```
.sort((a, b) => {
        const dateA = a.redeemed_at ? Date.parse(a.redeemed_at) : 0;
        const dateB = b.redeemed_at ? Date.parse(b.redeemed_at) : 0;
        if (dateA != dateB) {
          return dateB - dateA;
        }
        return b.sponsor_seller_id - a.sponsor_seller_id;
      })
```